integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/ratelimit/ApiLevelRatelimitTestCase.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.testcases.standalone.ratelimit;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.choreo.connect.tests.common.model.API;
+import org.wso2.choreo.connect.tests.common.model.ApplicationDTO;
+import org.wso2.choreo.connect.tests.util.ApictlUtils;
+import org.wso2.choreo.connect.tests.util.TestConstant;
+import org.wso2.choreo.connect.tests.util.TokenUtil;
+import org.wso2.choreo.connect.tests.util.Utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiLevelRatelimitTestCase {
+    private String testKey;
+
+    @BeforeClass
+    public void createApiProject() throws Exception {
+        ApictlUtils.createProject("ratelimit_openAPI.yaml", "ratelimit_test",
+                null, null, null, "ratelimit_api.yaml");
+        ApictlUtils.login("test");
+        ApictlUtils.deployAPI("ratelimit_test", "test");
+        Utils.delay(10000, "Could not wait till initial setup completion.");
```

Comment:
Shall we move the above section to where all the other APIs are created. That way we can get rid of the additional delay. When a number of APIs are deployed separated the delay also gets accumulated. Which is why we follow a pattern to deploy all at once.